src/components/Map/components/LegendDesc.tsx

```diff
@@ -9,9 +9,9 @@ const LegendDesc: Record<string, () => React.ReactElement> = {
       label="Nombre de lots d'habitation"
       color={themeDefEnergy.gas.color}
       scaleLabels={[
-        { label: '<\u00a0100', size: 0.5 },
-        { label: '100\u00a0à\u00a01000', size: 1 },
-        { label: '>\u00a01000', size: 2 },
+        { label: '<\u00a030', size: 0.8 },
+        { label: '30\u00a0à\u00a0100', size: 1.3 },
+        { label: '>\u00a0100', size: 2 },
```

Comment:
how do you correlate those 3 values (0.8, 1.3 and 2) with the one on the map (0.3, 0.5 and 0.85) ?

Reply:
I realize this LegendDesc (for Energy) is an old dead code. I clean this. 
But so, for a correlation with the legend icon or circle, it is based on a calc (with the Relative value em but here you got only the result of this calc - [elementFontSize, so 12px] * [scaleLabels.size] ). 
The calc is  carried by the ScaleLegend component.

Reply:
I added another commit for the update integration and the compute size system. It should be more clear. 😊